SPEC.md

```diff
@@ -1400,29 +1401,29 @@ task y {
 
 workflow.wdl
 ```
-import "tasks.wdl"
+import "tasks.wdl" as pyTasks
 
 workflow wf {
-  call x
-  call y
+  call pyTasks.x
+  call pyTasks.y
 }
 ```
 
-Tasks `x` and `y` are in the same namespace as workflow `wf` is.  However, if workflow.wdl could put all of those tasks behind a namespace:
+Tasks `x` and `y` are inside the namespace `pyTasks`, and which is different from the 'wf' namespace belonging to the primary workflow.  However, if no namespace is specified for tasks.wdl:
 
 workflow.wdl
 ```
-import "tasks.wdl" as ns
+import "tasks.wdl"
 
 workflow wf {
-  call ns.x
-  call ns.y
+  call tasks.x
+  call tasks.y
 }
 ```
 
-Now everything inside of `tasks.wdl` must be accessed through the namespace `ns`.
+Now everything inside of `tasks.wdl` must be accessed through the default namespace `tasks`.
 
-Each namespace contains: namespaces, tasks, and workflows.  The names of these needs to be unique within that namespace.  For example, there cannot be a task named `foo` and also a namespace named `foo`.  Also there can't be a task and a workflow with the same names, or two workflows with the same name.
+Each namespace contains: namespaces, tasks, and workflows.  The names of these needs to be unique within that namespace.  For example, there cannot be a workflow named `foo` and also an imported namespace named `foo`.  Also there can't be a workflow and a task in that workflow with the same names, or import two workflows with the same namespace. However, you can import two workflows with different namespace names that share all the same tasks. For example, you can import namespace 'foo' and 'bar', both of which contain a task 'baz', and you can call 'foo.baz' and 'bar.baz' inside the same primary workflow.
```

Comment:
For Miguel's third comment, I'd suggest "or two imported namespaces with the same names". 

TOL something like:
For example, there cannot be two imported workflows named `foo` in the same namespace, nor a workflow and a namespace both named `foo` in the same namespace. Similarly there cannot be a task `foo` in a workflow also named`foo`.

Question: is a workflow/namespace/task **in** a namespace, **has** a namespace, **named** a namespace, or something else? Using one verb consistently might help users conceptualize what the namespace is (and isn't).